doc/vim-manpager.txt

```diff
@@ -102,6 +103,31 @@ COMMAND					*vim-manpager-command*
 	under the cursor will be used.
 	It overwrite the default |:Man| command to improve the behavior.
 
+==============================================================================
+CONFIGURATION				*vim-manpager-configuration*
+
+
+					*g:manpager_default_keymaps*
```

Comment:
Please make it like `g:manpager_disable_default_mappings` and make it default `0`

Reply:
Thanks! I've forced push a commit to rename this option.